packages/web-components/fast-components/src/radio/radio.styles.ts

```diff
@@ -132,7 +132,7 @@ export const RadioStyles = css`
         cursor: ${disabledCursor};
     }
 
-    :host([checked]) .checked-indicator {
+    :host(.checked) .checked-indicator {
```

Comment:
could these use [aria-checked="true"] - just curious?

Reply:
yep prolly, I will try that